docs/source/reference-core.rst

```diff
@@ -768,43 +767,94 @@ inside the handler function(s) with the ``nonlocal`` keyword::
         async with trio.open_nursery() as nursery:
             nursery.start_soon(broken1)
 
-.. _strict_exception_groups:
+.. _handling_exception_groups:
+
+Designing for multiple errors
++++++++++++++++++++++++++++++
+
+Structured concurrency is still a young design pattern, but there are a few patterns
+we've identified for how you (or your users) might want to handle groups of exceptions.
+Note that the final pattern, simply raising an `ExceptionGroup`, is the most common -
+and nurseries automatically do that for you.
+
+**First**, you might want to 'defer to' a particular exception type, raising just that if
+there is any such instance in the group.  For example: `KeyboardInterrupt` has a clear
+meaning for the surrounding code, could reasonably take priority over errors of other
+types, and whether you have one or several of them doesn't really matter.
+
+This pattern can often be implemented using a decorator or a context manager, such
+as :func:`trio_util.multi_error_defer_to` or :func:`trio_util.defer_to_cancelled`.
+Note however that re-raising a 'leaf' exception will discard whatever part of the
+traceback is attached to the `ExceptionGroup` itself, so we don't recommend this for
+errors that will be presented to humans.
 
-"Strict" versus "loose" ExceptionGroup semantics
-++++++++++++++++++++++++++++++++++++++++++++++++
+..
+    TODO: what about `Cancelled`?  It's relevantly similar to `KeyboardInterrupt`,
+        but if you have multiple Cancelleds destined for different scopes, it seems
+        like it might be bad to abandon all-but-one of those - we might try to execute
+        some more code which then itself gets cancelled again, and incur more cleanup.
+
+        Maybe CancelScope itself should be ExceptionGroup-aware, and use .split() to
+        extract all and only matching Cancelled errors?  trio_util.defer_to_cancelled
+        only defers if there's exactly one Cancelled, which is a good intermediate step.
```

Comment:
Skimming the code, there seems to be some splitting going on? Idk if you're talking about some slightly different scenario though
https://github.com/python-trio/trio/blob/564907b15d8569b738047e4337f1ef3a2350cc30/src/trio/_core/_run.py#L596-L610

Reply:
No, I think I was just looking in the wrong place.  Will adjust the comment 👍